Update release notes for WAFS removal from GFS

KateFriedman-NOAA opened the pull request:
# Description

This PR updates the release notes for the WAFS removal from the GFS. Files and settings that are removed are noted.

Also update `gfs_ver=v16.3.19`.

@YaliMao-NOAA please let me know if the release notes are complete, especially the list of output files being removed, thanks!

Refs #2591

# Type of change

Production update

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO
- Does this change require an update to any of the following submodules? YES - WAFS is removed

KateFriedman-NOAA committed:
Update prepobs to v1.1.0 in GFSv16.3.16 (#2849)

Update the prepobs package version to `1.1.0` via the `prepobs_run_ver` variable in the version files.
This update was missed in the prior PR to update to `obsproc/v1.2` in GFSv16.3.16 but only affected
developer runs; operations is fine as it correctly gets prepobs/v1.1.0 through obsproc.

Resolves #2837
Refs #2840

KateFriedman-NOAA committed:
Updates for RTOFSv2.4.4 operational upgrade (#2800) (#2921)

The upstream RTOFS system was upgraded to v2.4.4 on September 13th, 2024.

A new GFS (`v16.3.17`) version resulted from this upgrade due to the
second digit change (`3 -> 4`) and resulting COM path update
(`../rtofs/v2.4`). No other change in the GFS in ops for this
operational upgrade.

Also some requested cleanup/decluttering of release notes. Have deleted
all older release notes and renamed the most recent version notes to
`Release_Notes.md`. Future implementations will update this same file
instead of making a new file.

Refs #2320

KateFriedman-NOAA committed:
Update release notes for WAFS removal from GFS

Also update gfs_ver=v16.3.19

Refs #2591

aerorahul reviewed (approved):
looks good.

YaliMao-NOAA commented on docs/Release_Notes.md:
```diff
@@ -49,75 +49,127 @@ cd ../ecf
 ------------
 
-* No changes from GFS v16.3.16
+All WAFS jobs are removed from the GFS ecFlow definition file, rocoto mesh, and `ush/ecflow/prod.yml`.
```
Should the jobs be listed here?

YaliMao-NOAA commented on docs/Release_Notes.md:
```diff
@@ -49,75 +49,127 @@ cd ../ecf
+* `gfs.tCCz.wafs.grb2f[000-120].idx`
+* `gfs.tCCz.wafs_grb45f[06-36].grib2`
+* `gfs.tCCz.wafs_grb45f[06-36].grib2.idx`
+* `WAFS_0p25_blended_YYYYMMDDCCf[06-48].grib2`
```
Some files are missing. Please refer column C in this document for a complete list: https://docs.google.com/spreadsheets/d/1Nt343Z9x9UycweFik3HRFpXkqIjs7m20s15yGOhsgUY

KateFriedman-NOAA committed:
GFSv16.3.18 - Increase the wait time for UKMet data in the WAFS blending_0p25 job (#2933)

The GFS is updated to v16.3.18 in operations to update the wait time for
UKMet data in the WAFS blending_0p25 job. Error handling is also updated
for late arriving UKMet data. Changes were provided by NCO. A new tag
was made in the WAFS repo.

Refs #2923

KateFriedman-NOAA commented:
@YaliMao-NOAA reminder to review this PR, thanks!

YaliMao-NOAA commented on docs/Release_Notes.md:
@KateFriedman-NOAA I left some comments on Monday. May you please take a look?

KateFriedman-NOAA replied:
@YaliMao-NOAA Thanks for the feedback! I have updated the release notes to add a list of the jobs removed and to add the missing files from the file list. Please review and let me know if anything else needs updating. Thanks!

YaliMao-NOAA commented:
@KateFriedman-NOAA As in the recent [WAFS PR#59](https://github.com/NOAA-EMC/WAFS/pull/59) , WAFS master file (old name: gfs.tCCz.wafs.grb2fFFF, new name: wafs.tCCz.master.fFFF.grib2) at forecast hour F001-F005 will be removed too.

KateFriedman-NOAA committed:
Merge remote-tracking branch 'origin/dev/gfs.v16' into release/gfs.v16.3.19

* origin/dev/gfs.v16:
  GFSv16.3.18 - Increase the wait time for UKMet data in the WAFS blending_0p25 job (#2933)
  Updates for RTOFSv2.4.4 operational upgrade (#2800) (#2921)
  Update prepobs to v1.1.0 in GFSv16.3.16 (#2849)

KateFriedman-NOAA committed:
Update release notes to mention new file name

Refs #2591

KateFriedman-NOAA commented:
@YaliMao-NOAA Thanks for the information. I already had that file listed in the release notes but have added text to state it is being renamed in the WAFS package. Please review and let me know if the release notes are good to go. Once I have your approval I will merge this and cut a hand-off tag. Thanks!

KateFriedman-NOAA committed:
Update release notes based on feedback from WAFS

Refs #2591

YaliMao-NOAA commented:
@KateFriedman-NOAA 
1. wmo/xtrn.wfsgfs0006.gfs_atmos_wafs_f30_00 is still in production.
2. Please add this line under the products to be retired:
wafs.tCCz.master.fFFF.grib2 where FFF is from 001 to 005

KateFriedman-NOAA committed:
Updates to file list in release notes

Refs #2591

KateFriedman-NOAA commented:
> 1. wmo/xtrn.wfsgfs0006.gfs_atmos_wafs_f30_00 is still in production.
> 2. Please add this line under the products to be retired:
wafs.tCCz.master.fFFF.grib2 where FFF is from 001 to 005

@YaliMao-NOAA updated @ 98b902b8

YaliMao-NOAA reviewed (approved):
Looks good.

KateFriedman-NOAA commented:
Thanks @YaliMao-NOAA ! I will now merge this into the auth repo release branch.
